pkg/lsp/handler.go

```diff
@@ -605,8 +605,13 @@ func (s *Server) Formatting(ctx context.Context, params *protocol.DocumentFormat
 	if err != nil {
 		return []protocol.TextEdit{}, nil
 	}
-
-	return []protocol.TextEdit{{Range: protocol.Range{End: protocol.Position{Line: math.MaxInt32}}, NewText: string(out)}}, nil
+    var length uint32 = 1
```

Comment:
can you replace with https://pkg.go.dev/strings#Count ?

Reply:
fixed. Thanks!